website/docs/plugin/framework/migrating/attributes-blocks/types.mdx

```diff
@@ -0,0 +1,96 @@
+---
+page_title: 'Attribute Types: Migrating from SDKv2 to the Framework'
+description: >-
+  Migrate attribute type from SDKv2 to the plugin Framework
+---
+
+# Attribute Types
+
+An attribute either contains a primitive type, such as an integer or a string, or contains other attributes. Attributes
+that contain other attributes are referred to as nested attributes, and are implemented by populating the
+`NestedAttributes` field on the `tfsdk.Attribute` struct. Refer to
+[Schemas - Attributes](/plugin/framework/schemas#type) in the Framework documentation for details.
```

Comment:
The examples in this page illustrate how to migrate a primitive type (`schema.TypeString` => `types.StringType`). 

I've added another example using a `schema.TypeList` => `types.ListType`.
Do you think that will suffice?

Reply:
I guess I had it in my mind that maybe we could show all the most common type migrations (bool, float64, int64, list, map, set, string), so folks can cross-reference every use case even if it is a little rote. I'm not sure these migration guide code examples necessarily need to relate to existing code examples from migrated providers (since they all will look very similar).

Reply:
I like the sound of that. Have updated the schema examples to reflect how the migration for the most common type migrations will look.